src/ansys_sphinx_theme/__init__.py

```diff
@@ -85,6 +85,29 @@ def setup_default_html_theme_options(app):
     app.config.html_theme_options.setdefault("collapse_navigation", True)
 
 
+def update_footer_theme(app: Sphinx, pagename: str, templatename: str, context, doctree) -> None:
```

Comment:
If possible, can you add the type of the `doctree` parameter.

Reply:
And ``context``